tests/misc.rs

```diff
@@ -1203,6 +1203,15 @@ test! {
   status:   EXIT_FAILURE,
 }
 
+test! {
+  name:     test_just_executable_function,
+  justfile: "a:\n  @echo Executable path is: {{just_executable()}}",
```

Comment:
You can use multi-line strings here. Also, I think the windows test failure is due to backslashes in the path being interpreted. Try this:

```rust
test! {
  name:     test_just_executable_function,
  justfile: "
    a:
      @echo Executable path is: '{{just_executable()}}'
    ",
```

(Multi-line strings aren't a feature of rust, they're from the test macro.)